src/screens/Schedule/ScheduleScreen.tsx

```diff
@@ -50,25 +66,119 @@ const ScheduleScreen = () => {
+        underlayColor={Colors.lightBlue}
+        style={{
+          backgroundColor: isActive ? Colors.lightBlue : undefined,
+        }}
+      >
+        <TextRegular
+          style={{
+            fontWeight: isActive ? 'bold' : undefined,
+          }}
+        >
+          {props.title}
         </TextRegular>
-      ))}
-    </PageContainer>
+      </SwitchButton>
+    );
+  };
+
+  const getSwitchDescription = () => {
+    if (switchPage === 0) {
+      // schedule new
+      return 'View any upcoming appointments you have with your attorney here. Reschedule or cancel your appointment through the confirmation email you received from Calendly.';
+    }
+    return 'Schedule appointments with attorney(s) for newly approved case(s) here.';
+  };
+
+  // convert datestring (appointment.startTime) and convert to readable string
+  const getDateString = (dateString: string) => {
+    // TODO: how do I best do this?
+    return dateString;
+  };
+
+  const getUpcomingBody = () => {
+    if (appointments === undefined || appointments?.length === 0) {
+      // if no upcoming appointments
+      return (
+        <View>
+          <TextRegular>
+            You have no upcoming appointments at this time. Check
+            <TextRegularBold> Schedule New</TextRegularBold> for any approved
+            consultations.
+          </TextRegular>
+        </View>
+      );
+    }
+    // need bottom <></> otherwise bleeds onto next screen
+    return (
+      <>
+        {appointments?.map((appointment, key) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <ApptContainer key={key}>
+            <TextSubtitle>{appointment.caseType}</TextSubtitle>
+            <TextRegular>You have an appointment scheduled for </TextRegular>
+            <TextRegularBold>
+              {getDateString(appointment.startTime)}
+            </TextRegularBold>
+            <TextRegular>.</TextRegular>
+          </ApptContainer>
+        ))}
+        <></>
+      </>
+    );
+  };
+
+  const getScheduleBody = () => {
+    if (calendlyLinks === undefined || calendlyLinks?.length === 0) {
+      // if not approved for appointments
+      return (
+        <View>
+          <TextRegular>
+            You have not been approved for appointments in any new cases. Check{' '}
+            <TextRegularBold>Home</TextRegularBold> for any missing items.
+          </TextRegular>
+        </View>
+      );
+    }
+    return (
+      <>
+        <TextRegular>You can now schedule appointments for:</TextRegular>
+        <ScheduleContainer>
+          {calendlyLinks?.map(cl => (
+            <ScheduleButton
+              onPress={() => openCalendlyInBrowser(cl.link)}
+              key={cl.link}
+            >
+              <TextSubtitle>{cl.type} </TextSubtitle>
+              <MaterialCommunityIcons
+                name="open-in-new"
+                color={Colors.lightGray}
+                size={26}
+              />
+            </ScheduleButton>
```

Comment:
This is erroring because ScheduleButton can only have one child component. It worked when I wrapped the children with `<>...</>`
![screen_shot_2021-11-28_at_2 18 21_am](https://user-images.githubusercontent.com/55935661/143784779-d4d9b9e4-775a-453f-a549-41897b511786.png)

Reply:
yeah - i think that was a residual error from shannon's commit, just fixed